ArduCopter/Parameters.cpp

```diff
@@ -969,6 +969,15 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     AP_SUBGROUPPTR(mode_zigzag_ptr, "ZIGZ_", 38, ParametersG2, ModeZigZag),
 #endif
 
+#if MODE_ACRO_ENABLED == ENABLED
+    // @Param: ACRO_OPTIONS
+    // @DisplayName: Acro mode options
+    // @Description: A range of options that can be applied to change acro mode behaviour. Air-mode enables ATC_THR_MIX_MAN at all times. Rate loop disables the use of angle stabilization and uses the rate loop only.
```

Comment:
I would suggest adding a statement that makes it clear for tradheli users that Air-mode is multicopter only.  maybe even put it in parenthesis in the bitmask list below so it looks like this.
@Bitmask: 0:Air-mode(multicopter only),1:Rate Loop

Reply:
and Rate Loop only?

Reply:
Ok I've fixed up in  the docs. Air-mode has no effect on helis as they are always stabilizing, so I figured it was better to just say that in the docs. Ok?